examples/AspNetCore/Controllers/WeatherForecastController.cs

```diff
@@ -45,6 +48,11 @@ public IEnumerable<WeatherForecast> Get()
         // how dependency calls will be captured and treated
         // automatically as child of incoming request.
         var res = HttpClient.GetStringAsync("http://google.com").Result;
+
+        // Manually create an activity. This will become a child of
```

Comment:
lets also state in the comment that, this is really an optional step, and most users should be only requiring the activity created by the instrumentation library for aspnetcore. 
If the user's goal is to add more stuff to the activity, show that they should simply use Activity.Current?.SetTag.
If the user's goal is to wrap something worthy of tracking as a separate activity, then only they need to manually create a new activity. 
We do not want every users to copy this  as-is, and end up with more activity (more perf hit + transmission/storage costs), than actually needed.

Reply:
Thoughts on the following? Do you think we need a note for the counter?
```csharp
// Optional: Manually create an activity. This will become a child of
// the activity created from the instrumentation library for AspNetCore.
// Manually created activities are useful when there is a desire to track
// a specific subset of the request. In this example one could imagine
// that calculating the forecast is an expensive operation and therefore
// something to be distinguished from the overall request.
// Note: Tags can be added to the current activity without the need for
// a manual activity using Acitivty.Current?.SetTag()
using var activity = this.instrumentation.ActivitySource.StartActivity("calculate forecast");
```

Reply:
both looks good